app/services/description_validator.rb

```diff
@@ -40,12 +40,13 @@ def validate_duplicate_headers
   end
 
   def validate_title_headers
-    if @headers.include?('title1.value') ||
-       (@headers.exclude?('title1.value') && @headers.include?('title1.structuredValue1.type') && @headers.include?('title1.structuredValue1.value'))
-      return
-    end
+    title_count = 0
+    title_count += 1 if title_value_header?
+    title_count += 1 if title_structured_value_header?
+    title_count += 1 if title_parallel_value_header?
 
-    errors << 'Title column not found.'
+    errors << 'Title column not found.' if title_count.zero?
+    errors << 'Multiple title columns found.' if title_count > 1
```

Comment:
I believe multiple title columns is valid. Consider one row has a parallel title and one row has a structured title.

Reply:
That makes sense. The previous code prevented it (notice the `exclude`) so I assumed that was intentional. Removing this check.